python/python/glide/async_commands/core.py

```diff
@@ -2662,6 +2664,72 @@ async def zmscore(
             await self._execute_command(RequestType.ZMScore, [key] + members),
         )
 
+    async def zinterstore(
+        self,
+        destination: str,
+        keys: Union[List[str], List[Tuple[str, int]]],
+        aggregation_type: Optional[AggregationType] = None,
+    ) -> int:
+        """
+        Computes the intersection of sorted sets given by the specified `keys` and stores the result in `destination`.
```

Comment:
I suggest simplifying the doc. You can copy it from [`sinterstore`](https://github.com/aws/glide-for-redis/pull/1278) or from [`zdiffstore`](https://github.com/aws/glide-for-redis/pull/1378).

Line
> If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.

Is true for all write commands, we don't mention that for others (e.g. sadd, zadd, lpush, set and so on).

Reply:
I copied the doc from the Java impl. for this command.
https://github.com/aws/glide-for-redis/blob/main/java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java#L846

Reply:
Oh I see. Hmm, we need to discuss whether we need that.